### PR TITLE
Usermodels

### PR DESCRIPTION
**Pull request by tyasvdspree**

### CODE REVIEW

**Review by MrTheGood — CHANGES_REQUESTED** [review]
Resolve the conflicts please

File: src/components/Single/SidePanel/Timeline.vue [hunk 1]
```diff
@@ -86,46 +91,41 @@
...
-        background-color: #45C879;
-
-        box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
-    }
+        .createPost .FAB {
+            display: flex;
+            justify-content: center;
+            width: 56px;
+            height: 56px;
+            border-radius: 50%;
```
Comment (MrTheGood) [inline src/components/Single/SidePanel/Timeline.vue]: this is going to cause some conflicts

File: src/components/Single/SidePanel/Timeline.vue [hunk 2]
```diff
@@ -1,56 +1,61 @@
...
-            </div>
-
-            <div class="FABItem">
-                <p>New trip</p>
-                <img src="img/ic_trip_start.svg"/>
-            </div>
-
-            <div class="FAB">
-                <img src="img/ic_add.svg"/>
-            </div>
```
Comment (MrTheGood) [inline src/components/Single/SidePanel/Timeline.vue:41]: Why did this get removed?

Comment (MrTheGood) [reply src/components/Single/SidePanel/Timeline.vue:41]: This definitely should not get removed

Comment (MrTheGood) [reply src/components/Single/SidePanel/Timeline.vue:41]: Screenshot 2019-12-09 at 16 00 55
it's this

**Review by CJVisser — APPROVED** [review]

**Review by CJVisser — APPROVED** [review]

**Review by MrTheGood — APPROVED** [review]

**Pull request merged**